hassio-google-drive-backup/backup/static/layouts/base.jinja2

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {% block head %}
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0" />
+  <title>Home Assistant Google Drive Backup</title>
+
+  <link href="static/css/icons.css?version={{ version }}" rel="stylesheet">
+  <link href="static/css/materialize.css?version={{ version }}" type="text/css" rel="stylesheet" media="screen,projection" />
+  <link href="static/css/colpick.css?version={{ version }}" type="text/css" rel="stylesheet" media="screen,projection" />
+  <link href="static/css/static.css?version={{ version }}" type="text/css" rel="stylesheet" media="screen,projection" />
+  <link href="https://fonts.googleapis.com/css?family=Cookie" rel="stylesheet">
+  <link rel="icon" type="image/png" href="static/images/favicon.png?version={{ version }}">
+
+  <style id="theme" type="text/css" data-background-color="{{ backgroundColor }}" data-accent-color="{{ accentColor }}">
```

Comment:
From my [last comment](https://github.com/sabeechen/hassio-google-drive-backup/pull/248/files#r479875236), the colors are sent from the base context directly to this tag.